### PR TITLE
adding CICD env variable example

### PR DESCRIPTION
**Pull request by anaxceron**
Describe your pull request here: adding do re: using environment variables in CI/CD pipelines. resolves [Issue 3504](https://github.com/zowe/docs-site/issues/3504).

List the file(s) included in this PR: cli-using-environment-variables.md

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

**Review by t1m0thyj — APPROVED** [review]

**Review by adam-wolfe — APPROVED** [review]
LGTM

File: docs/user-guide/cli-configure-scs-on-headless-linux-os.md [hunk 1]
```diff
@@ -2,12 +2,22 @@
 
 Perform the following configurations on headless and z/Linux operating systems.
 
+:::note
+
+For CI/CD pipelines we recommend using the credential management provided by the CI/CD tool (for example, Jenkins credentials or GitHub secrets) to store credentials and load them into environment variables in the pipeline. See [Using environment variables](../user-guide/cli-using-using-environment-variables.md#store-credentials-securely-in-cicd-pipelines) for more information.
```
Comment (sbohnert) [inline docs/user-guide/cli-configure-scs-on-headless-linux-os.md:7]: Add comma
For CI/CD pipelines, we recommend

**Review by sbohnert — COMMENTED** [review]
Looks good.

File: docs/user-guide/cli-configure-scs-on-headless-linux-os.md (hunk 1 above)
Comment (anaxceron) [reply docs/user-guide/cli-configure-scs-on-headless-linux-os.md:7]: addressed in [PR 3580](https://github.com/zowe/docs-site/pull/3580)

**Pull request merged**